contrib/templates/stats.html.tmpl

```diff
@@ -26,7 +26,7 @@ Taken at {{ .Now }}
 
 <h3 id="gossip-peers">Gossip Peers</h3>
 <table><tr><th>Name</th><th>HTTP</th><th>Recon</th><th>Recon Status</th><th>Recovery Status</th></tr>
-{{ range $peer := .Peers }}<tr><td>{{ $peer.Name }}</td><td><a href="http://{{ $peer.HTTPAddr }}/pks/lookup?op=stats">{{ $peer.HTTPAddr }}</a></td><td>{{ $peer.ReconAddr }}</td><td>{{ $peer.ReconStatus }}</td><td>{{ $peer.RecoveryStatus }}</td></tr>
+{{ range $peer := .Peers }}<tr><td>{{ $peer.Name }}</td><td><a href="http://{{ $peer.HTTPAddr }}{{ $peer.StatsURI }}">{{ $peer.HTTPAddr }}</a></td><td>{{ $peer.ReconAddr }}</td><td>{{ $peer.ReconStatus }}</td><td>{{ $peer.RecoveryStatus }}</td></tr>
```

Comment:
If StatsURI is being appended to HTTPAddr, maybe it should be called StatsPath instead, since "URI" implies that it contains a full URI?

Reply:
Ahh... yes. I was misunderstanding URI to be a subset of URL, which is actually the reverse.